### ISSUE TITLE
The demo page is broken

### ISSUE THREAD

**Issue by lilianalillyy** [mozilla/sanitizer-polyfill#294]
The page is trying to load the polyfill script at https://mozilla.github.io/sanitizer-polyfill/dist/sanitizer-polyfill.js, however the file is not available there anymore, causing the whole demo to be completly broken.

**Comment by mozfreddyb** [comment]
Thank you for filing this issue. The polyfill is indeed in a bit of a sorry state: The spec is under heavy development is undergoing some major breaking changes.

I hope to find some time when the spec has matured a bit further.

Until then, I'm grateful for people helping out with minor quirks and issues (like the path issue you describe) in pull requests.

**Comment by lilianalillyy** [comment]
> Thank you for filing this issue. The polyfill is indeed in a bit of a sorry state: The spec is under heavy development is undergoing some major breaking changes.
> 
> I hope to find some time when the spec has matured a bit further.
> 
> Until then, I'm grateful for people helping out with minor quirks and issues (like the path issue you describe) in pull requests.

Yeah, I've been watching it closely, I personally have been using the polyfill on few of my sites (just ported to typescript), it works amazing and better than any solution I've used before, so I'm looking forward to see the spec develop further :)
